pinecone_text/sparse/splade_encoder.py

```diff
@@ -1,6 +1,20 @@
 from typing import List, Union
 
-import torch
+try:
+    import torch
+except (OSError, ImportError) as e:
+    raise ImportError(
+        "Failed to import torch. Make sure you install pytorch extra dependencies by running:"
+        "pip install pinecone-text[torch] ."
+        "If this doesn't help, it is probably a CUDA error. "
+        "If you do want to use GPU, please check your CUDA driver. "
+        "If you want to use CPU only, run the following: "
+        "`poetry run pip uninstall -y torch torchvision; "
```

Comment:
```suggestion
        "`pip uninstall -y torch torchvision; "
```
The users don't need `poetry run`. They're installing your code from PYPI, not with `poetry install`